gcc/rust/ast/rust-ast.h

```diff
@@ -884,6 +884,15 @@ class MetaListNameValueStr;
 class Stmt : public Node
 {
 public:
+  enum class Kind
+  {
```

Comment:
Brilliant more of this please :D. Everyone hates visitors and more of this will allow use to use switch statements and write a function something like the GCC walk_tree eventually.